portal/src/graphql/portal/BotProtectionConfigurationScreen.tsx

```diff
@@ -964,31 +1116,26 @@ const BotProtectionConfigurationContentRequirementsSection: React.VFC<BotProtect
           <WidgetTitle>
             <FormattedMessage id="BotProtectionConfigurationScreen.requirements.title" />
           </WidgetTitle>
-          <Text as="h3" block={true} className={styles.subtitle}>
-            <FormattedMessage id="BotProtectionConfigurationScreen.requirements.flows.subtitle" />
-          </Text>
-          <ChoiceGroup
-            selectedKey={requirements.flows.flowType}
-            options={flowOptions}
-            onChange={onChangeFlowType}
-          />
         </div>
         <div>
-          <HorizontalDivider />
+          <BotProtectionConfigurationContentRequirementsSectionFlowHeader
+            requirements={requirements}
+            setRequirements={setRequirements}
+          />
           <DetailsList
+            // FIXME: on hover background color should not change
```

Comment:
Any suggestion on overriding background color? Tried below both not work

1. `DetailsList.theme={{palette: {neutralLighter: "#0"}}`
2. `DetailsList.styles={root:{":hover"}}`

Reply:
Check the source code and see how to control it?
https://github.com/microsoft/fluentui/blob/b5e427c24efd04fbd54dd4e43819ddc24be0ac16/packages/react/src/components/DetailsList/DetailsRow.styles.ts#L315